src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,69 @@
 package core.basesyntax;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
 public class WorkWithFile {
+    public static final String COMMA_CHAR = ",";
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
+    public static final String RESULT = "result";
+    public static final int AMOUNT = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] fileContent = readFromFile(fromFileName);
+        String report = writeReport(fileContent);
+        writeToFile(report, toFileName);
+    }
+
+    private String[] readFromFile(String fromFileName) {
+        Path fromFilePath = Paths.get(fromFileName);
+        try {
+            List<String> lines = Files.readAllLines(fromFilePath);
+
+            StringBuilder stringBuilderContent = new StringBuilder();
+            for (String line : lines) {
+                stringBuilderContent.append(line).append("\n");
+            }
+            String stringBuilder = stringBuilderContent.toString();
+            return stringBuilder.split(COMMA_CHAR + "|\\n");
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read a file " + fromFileName, e);
+        }
+    }
```

Comment:
in this method you should only read, parse or formatting place in create report method